packages/orchestrator-ui-components/src/components/WfoTable/WfoTable/WfoMultilineCell.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { FC, ReactElement } from 'react';
+
+export type WfoMultilineCellProps = {
+    children?: ReactElement | string | null;
```

Comment:
Is use the ReactNode for children, that includes string, number etc.. What is the benefit of ReactElement here?

Reply:
ReactNode  is throwing compile errors that it is not allowed in an FC

Reply:
Looks a bit weird that children is plural, but we got an Element or a string or null. I was expecting a list.